aws/resource_aws_rds_cluster_test.go

```diff
@@ -364,6 +364,32 @@ func TestAccAWSRDSCluster_s3Restore(t *testing.T) {
 	})
 }
 
+func TestAccAWSRDSCluster_PointInTimeRestore(t *testing.T) {
+	var v rds.DBCluster
+	var c rds.DBCluster
+
+	parentId := acctest.RandomWithPrefix("tf-acc-point-in-time-restore-seed-test")
+	restoredId := acctest.RandomWithPrefix("tf-acc-point-in-time-restored-test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSClusterConfig_pointInTimeRestoreSource(parentId, restoredId),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSClusterExists("aws_rds_cluster.test", &v),
+					resource.TestCheckResourceAttr("aws_rds_cluster.test", "engine", "aurora-mysql"),
+					testAccCheckAWSClusterExists("aws_rds_cluster.restored_pit", &c),
+					resource.TestCheckResourceAttr("aws_rds_cluster.restored_pit", "cluster_identifier", restoredId),
+					resource.TestCheckResourceAttr("aws_rds_cluster.test", "engine", "aurora-mysql"),
```

Comment:
L386 duplicates L382..maybe "aws_rds_cluster.restored_pit" was meant in place? you could potentially change L386 or converge the 2 checks to one with `resource.TestCheckResourcePairAttr()`
```suggestion
					testAccCheckAWSClusterExists("aws_rds_cluster.test", &v),
					testAccCheckAWSClusterExists("aws_rds_cluster.restored_pit", &c),
					resource.TestCheckResourceAttr("aws_rds_cluster.restored_pit", "cluster_identifier", restoredId),
					resource.TestCheckResourceAttrPair("aws_rds_cluster.restored_pit", "engine", "aws_rds_cluster.test", "engine"),

```